src/components/v2/shared/DistributionSplitsSection/DistributionSplitModal.tsx

```diff
@@ -0,0 +1,381 @@
+    current && current < moment().endOf('day')
+
+  const distributionLimitIsInfinite =
+    !distributionLimit || parseWad(distributionLimit).eq(MAX_DISTRIBUTION_LIMIT)
+
+  return (
+    <Modal
+      title={mode === 'Edit' ? t`Edit existing split` : t`Add a split`}
+      visible={visible}
+      onOk={setSplit}
+      okText={mode === 'Edit' ? t`Save split` : t`Add split`}
+      onCancel={onClose}
+      destroyOnClose
+    >
+      <Form
+        form={form}
+        layout="vertical"
+        onKeyDown={e => {
+          if (e.key === 'Enter') setSplit()
+        }}
+      >
+        <Form.Item>
+          <Select value={editingSplitType} onChange={setEditingSplitType}>
+            <Select.Option value="address">
+              <Trans>Wallet address</Trans>
+            </Select.Option>
+            <Select.Option value="project">
+              <Trans>Juicebox project</Trans>
+            </Select.Option>
+          </Select>
+        </Form.Item>
+
+        {editingSplitType === 'address' ? (
+          <FormItems.EthAddress
+            name="beneficiary"
+            defaultValue={beneficiary}
+            formItemProps={{
+              label: t`Address`,
+              rules: [
+                {
+                  validator: validatePayoutAddress,
+                },
+              ],
+            }}
+            onAddressChange={(beneficiary: string) =>
+              setBeneficiary(beneficiary)
+            }
+          />
+        ) : (
+          <Form.Item
+            name={'projectId'}
+            rules={[{ validator: validateProjectId }]}
+            label={t`Project ID`}
+            required
+          >
+            <InputNumber
+              value={parseInt(projectId ?? '')}
+              style={{ width: '100%' }}
+              placeholder={t`ID`}
+              onChange={(projectId: number) => {
+                setProjectId(projectId?.toString())
+              }}
+            />
+          </Form.Item>
+        )}
+        {editingSplitType === 'project' ? (
+          <FormItems.EthAddress
+            name="beneficiary"
+            defaultValue={beneficiary}
+            formItemProps={{
+              label: t`Token beneficiary`,
+              extra: t`The address that should receive the tokens minted from paying this project.`,
+            }}
+            onAddressChange={beneficiary => {
+              setBeneficiary(beneficiary)
+            }}
+          />
+        ) : null}
+
+        {/* Only show amount input if project distribution limit is not infinite */}
+        {!distributionLimitIsInfinite ? (
+          <Form.Item
+            label={t`Amount`}
+            className="ant-form-item-extra-only"
+            extra={
+              feePercentage && percent && !(percent > 100) ? (
+                <>
+                  {editingSplitType === 'address' ? (
+                    <div>
+                      <Trans>
+                        Payee will receive{' '}
```

Comment:
**suggestion**: maybe for brevity/clarity it should be:

```suggestion
```

And we add the `(?)` icon with a tooltip:

`13ETH after 2.5% JBX fee (?)`

The tooltip says something like:

`Payouts to Ethereum addresses incur a 2.5% fee. Your project will receive JBX in return at the current issuance rate.`

Reply:
Yeah love it